zha/application/platforms/sensor/__init__.py

```diff
@@ -686,23 +683,17 @@ def __init__(
         super().__init__(cluster_handlers, endpoint, device, **kwargs)
         self._attr_extra_state_attribute_names: set[str] = {
             "measurement_type",
-            self._max_attribute_name,
+            self._attr_max_attribute_name,  # None values are ignored in HA
         }
 
-    @property
-    def _max_attribute_name(self) -> str:
-        """Return the max attribute name."""
-        return self._attr_max_attribute_name or f"{self._attribute_name}_max"
```

Comment:
Explanation:
If a `_max` extra state attribute is wanted for an entity, it has to be explicitly specified using `_attr_max_attribute_name` now. This is done for all existing sensors, except the recently added "total active power" sensor, where the inclusion was inadvertent, as we do not even poll that `_max` attribute.